internal/printer/print-to-js.go

```diff
@@ -241,7 +241,8 @@ func render1(p *printer, n *Node, opts RenderOptions) {
 	case CommentNode:
 		p.addSourceMapping(n.Loc[0])
 		p.print("<!--")
-		p.print(n.Data)
+		backticks := regexp.MustCompile("`")
+		p.print(backticks.ReplaceAllString(n.Data, "\\`"))
```

Comment:
Actual fix was easy! Just copied from above

Reply:
Nice! Would be good to maybe extract this to an `escape` function?